etsin_finder/frontend/static/index.template.ejs

```diff
@@ -10,6 +10,7 @@
     />
     <meta name="theme-color" content="#007FAD" />
     <link rel="manifest" href="/static/manifest.json" />
+    <link href="https://fonts.googleapis.com/css?family=Lato:300,400,700&display=swap" rel="stylesheet">
```

Comment:
should we host the font on the server as this might cause firewall issues to end users.

Reply:
It is better to host this on the server, good suggestion.